modules/royalty/royalty-util.js

```diff
@@ -0,0 +1,65 @@
+const formattedMemberName = (member) => {
+  return `${member.get('name')}, ${member.get('title')}`
+}
+
+module.exports = class RoyaltyHelper {
```

Comment:
Helper --> Util.

Doesn't matter unless it's an error and it will throw the class name